package.json

```diff
@@ -2,24 +2,23 @@
   "name": "react-ssr-prepass",
   "version": "1.3.0",
   "description": "A custom partial React SSR renderer for prefetching and suspense",
-  "main": "index.js",
+  "main": "dist/react-ssr-prepass.js",
+  "module": "dist/react-ssr-prepass.es.js",
+  "types": "dist/react-ssr-prepass.d.ts",
```

Comment:
Should we also add import-maps?

```
exports: {
  '.': { import, require}
}
```

Reply:
We can't add export maps, since that may put Node/Webpack into weird modes again where they wouldn't accept the React child dependency afaik